ec2/spark_ec2.py

```diff
@@ -145,6 +145,14 @@ def parse_args():
         default=DEFAULT_SPARK_GITHUB_REPO,
         help="Github repo from which to checkout supplied commit hash (default: %default)")
     parser.add_option(
+        "--spark-ec2-git-repo",
+        default=DEFAULT_SPARK_EC2_GITHUB_REPO,
+        help="Github repo from which to checkout spark-ec2 (default: %default)")
+    parser.add_option(
+        "--spark-ec2-branch",
```

Comment:
I'll do the former so that it's consistent with the existing --spark-git-repo arg